VirtoCommerce.Storefront.Tests/Scriban/ScribanTests.cs

```diff
@@ -20,91 +20,110 @@ public class ScribanTests
         /// https://github.com/lunet-io/scriban/issues/102
         /// </summary>
         [Fact]
-        public void LiquidRawBlock_ParsingError()
+        public void Parse_EmptyRawBlock_IsCorrect()
         {
-            var parsedTemplate = Template.ParseLiquid("{% raw %}{% endraw %}{{ some_var }}");
+            // arrange
+            var template = "{% raw %}{% endraw %}{{ some_var }}";
+            // act
```

Comment:
there should be an empty space after comment line, e.g:

// comment 1
var code ...... some code;

// comment 2
var code2 .... some code;

this approach is recommended for better readability

Reply:
@t13ka fixed